lib/muffin_man/finances/v0.rb

```diff
@@ -15,7 +16,10 @@ def list_financial_event_groups(max_results_per_page = nil, financial_event_grou
         end
         @query_params["NextToken"] = next_token unless next_token.nil?
         @request_type = "GET"
-        call_api
+        res = call_api
```

Comment:
can we add this in call_api so that any api calls can use this and add enableLogging module wise?

Reply:
done